### ISSUE TITLE
Feature request: optional autostart, defined from env variable

### ISSUE THREAD

**Issue by sielaq** [Supervisor/supervisor#575]
I'm runing supervisord in docker container.
I would like to have multiple apps defined,
but some of them automatically started only if ENV variable allows for that
like:
```
# container started with 'docker run -e FOO_START="false" ...'

[program:foo]
command=bar
autostart=%(ENV_FOO_START)s
```
currently it is not possible 

```
Error: not a valid boolean value: '%(ENV_FOO_START)s'
```
also when variable is empty it supposed to take default, whcih is "true"

How do you think, sounds reasonable ?

**Comment by mnaberez** [comment]
This will be included in Supervisor 3.2.  It was fixed in 2d6ca34582a8a07a5dd96ae45ef62cd58a459f4f and there is a [unit test](https://github.com/Supervisor/supervisor/blob/edd4a734fda38e01e7f35ce0ba6d5d7ce03494db/supervisor/tests/test_options.py#L1008) for it.

**Comment by sielaq** [comment]
thx man, sorry for duplicate
